frontend/src/features/permits/components/dashboard/ApplicationStepPage.tsx

```diff
@@ -141,7 +138,12 @@ export const ApplicationStepPage = ({
     );
   };
 
-  if (isInvalidApplication || !isValidApplicationStatus() || !companyId || !isPermitTypeAllowed()) {
+  if (
+    isInvalidApplication ||
+    !isValidApplicationStatus() ||
+    !companyId ||
+    !isPermitTypeAllowed()
+  ) {
     return <Navigate to={ERROR_ROUTES.UNEXPECTED} />;
```

Comment:
Can you add a console statement?  makes life easier for debugging.
`console.error('The application cannot be displayed or edited');`